lib/index.js

```diff
@@ -209,14 +189,16 @@ module.exports = class Breakpoints {
    * Generate, format and write the CSS file
    */
   #generateAndWriteCSS() {
-    const filePath = path.resolve(process.cwd(), this.#userConfig.paths.css);
+    const filePath = path.resolve(process.cwd(), this.#userConfig.css.path);
     let string = "";
     this.#customProperties.forEach((customProperty) => {
       string += `--${customProperty.identifier}: "${customProperty.value}";`;
     });
     Breakpoints.writeFile(
       filePath,
-      prettier.format(`html {${string}}`, { parser: "css" }),
+      prettier.format(`${this.#userConfig.css.element} {${string}}`, {
+        parser: "css",
+      }),
```

Comment:
Idea for future expansion: Make Prettier configurable, so package users can switch to tabs, single quotes, whatever.